lib/govuk_index/publishing_event_worker.rb

```diff
@@ -39,9 +39,11 @@ def perform(messages)
         process_action(processor, routing_key, payload)
       end
 
-      response = processor.commit
-      process_response(response, messages) if response
+      responses = processor.commit
```

Comment:
Why does this now return multiple responses?  It doesn't look like the `ElasticsearchProcessor` has changed

Reply:
The client (ElasticsearchProcessor.new.govuk) is a subclass of `lib/index/client.rb` which now performs write actions against all active clusters:

```
    def bulk(params)
      Clusters.active.map do |cluster|
        client(cluster: cluster).bulk(
          params.merge(index: index_name)
        )
      end
    end
```

When `commit` is called on the processor (using the client to make the requests), it calls the modified bulk method:

```
@client.bulk(
        body: @actions
      )
```

Sorry, it's a bit convoluted. Let me know if I have misunderstood how this works.

Reply:
Ok, that makes sense.